.github/workflows/uml.yml

```
@@ -35,10 +35,11 @@ jobs:
 
       - name: Push up the changes
         run: |
-          git add docs/source/uml/*.png
+          git add docs/source/_static/*.png
```

Shoot. Not this change. 
Sorry copy and paste
```suggestion
          git add docs/source/uml/*.png
```